### PR TITLE
TOC meeting 2022-09-07

### PR DESCRIPTION
**Pull request by msvticket**
Renamed notes to make the default sorting in GitHub chronological
Converted agenda to meeting minutes
Updated README to reflect non-bootstrap status

### CODE REVIEW

File: TOC/README.md [hunk 1]
```diff
@@ -1,68 +1,63 @@
 # Technical Oversight Committee (TOC)
 
-## Initial Bootstrap Committee
+## Chair ##
 
-The Initial bootstrap committee had to be replaced by a new bootstrap committee.
-This bootstrap committee should be replaced by an elected Jenkins X TOC in the future.
+Current chair is Mårten Svantesson [@msvticket](https://github.com/msvticket), elected in July 6:th 2022.
```
Comment (ankitm123) [inline TOC/README.md:5]: extra `:` between 6 and th? does not render as superscript, if that was the intention 🤔

**Review by ankitm123 — APPROVED** [review]
LGTM

File: TOC/README.md (hunk 1 above)
Comment (msvticket) [reply TOC/README.md]: Fixed

**Pull request merged**